README.md

```diff
@@ -25,7 +25,6 @@ You might have to enable some extensions in `php.ini` (see the `ext-*` requireme
 ```
 extension=curl
 extension=gmp
-extension=mysqli
```

Comment:
Is it premature to remove this from the readme?

Reply:
Probably, yes. When I started this PR I was of the impression that this should all be changed in one fell swoop so the mysqli requirement can be dropped right away. A wild misjudgement of this undertaking's scale, again 😅
I'll revert this change for now - removing mysql everywhere is another few hours of work away.